src/Components/test/testassets/BasicTestApp/SectionsTest/ParentComponentWithTwoChildren.razor

```diff
@@ -0,0 +1,97 @@
+﻿@using Microsoft.AspNetCore.Components.Sections
+
+<h1>Parent Component</h1>
+
+@if (SectionOutletExists)
+{
+    <SectionOutlet Name="@firstSectionName" />
+}
+
+<p>Text between two section outlets</p>
+
+<SectionOutlet Name="@secondSectionName" />
+
+<button id="section_outlet_dispose"
```

Comment:
Nit: We generally use hyphens ("-") instead of underscores ("_") for `id` and `class` HTML attribute values.